docs/csharp/tutorials/exploration/records.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Use record types - C# tutorial
```

Comment:
f1 keyword metadata?

Reply:
I doubt it should be here, may be for some time only. It's not the language reference page. Ideally, I expect F1 to bring me to the language reference page about the `record` keyword or record types in general.